### DISCUSSION TITLE
SPDM 1.3 Work-in-Progress Draft Release

### DISCUSSION THREAD

**Discussion by steven-bellock** [DMTF/libspdm#1911] [Announcements]
The nearly complete SPDM 1.3 specification has been released at https://www.dmtf.org/sites/default/files/standards/documents/DSP0274_1.3.0WIP99.pdf as a work-in-progress draft.

**Comment by hiption** [comment]
Can I comment SPDM 1.3 in this page?

**Reply by steven-bellock** [reply]
Would be best to take questions / comments to https://spdmforum.com/

**Comment by steven-bellock** [comment]
Closing as the 1.3 specification has been released.
